api/GlobalEventHandlers.json

```diff
@@ -2899,7 +2899,7 @@
             ],
             "firefox_android": [
               {
-                "version_added": "59"
+                "version_added": "79"
```

Comment:
Fixes all the `onpointerxx` variants behind `dom.w3c_pointer_events.enabled` from 59 to 79